scripts/network_bandwidth.sh

```diff
@@ -2,7 +2,18 @@
 
 INTERVAL="1"  # update interval in seconds
 
-network_name=$(tmux show-option -gqv "@tmux2k-network-bandwidth")
+# Network interface to monitor
+network_name="en0"
```

Comment:
Hardcoding the network name doesn't feel like a good idea, we can use network name passed in as an option for better usability

Reply:
so I thought that in mac or any laptop, people mostly use wifi to connect to the internet, and this `en0` is the wifi. But, I will change it.

Reply:
I'm merging this in, will add the name option and docs

Reply:
Okay